accord-core/src/main/java/accord/messages/PreAccept.java

```diff
@@ -121,7 +142,7 @@ default MessageType type()
             return MessageType.PREACCEPT_RSP;
         }
 
-        boolean isOK();
+        boolean isOk();
     }
 
     public static class PreAcceptOk implements PreAcceptReply
```

Comment:
I decided no, to avoid confusion with situations where PartialDeps represent a complete set of deps for a range (in this case, they are being assembled). We could introduce a new concept _like_ {{PartialDeps}} to represent this situation, but I'm not sure it's super valuable.

Reply:
I don't understand why `PartialDeps` would be confusing. `Deps` is just `PartialDeps` for all ranges and both `Deps` and `PartialDeps` can be incomplete in terms of containing all the dependencies since they must be collected.

What would be nice is if `PartialDeps` and `Deps` were subclasses of `AbstractDeps`, sure a `PartialDeps` can be converted to a `Deps` through a checked conversion in some cases, but that is pretty specialized and it is obvious when this conversion is occurring.

In a lot of cases we know we are working with `PartialDeps` or `Deps`?

Maybe the distinction isn't very valuable, but the ambiguity of `Deps` vs `PartialDeps` might be worth eliminating.

Don't feel super strongly about this, but in the early stages of onboarding when you don't understand the implementation and protocol it's not always so obvious what kind of deps you are looking at.

Reply:
I’m sympathetic to this view. Perhaps we can explore it a little in routables? That’s largely the thrust of that patch already, just in adjacent areas.